src/Tickets/Emails/JSON_LD/Handler.php

```diff
@@ -0,0 +1,103 @@
+<?php
+namespace TEC\Tickets\Emails\JSON_LD;
+
+use TEC\Tickets\Commerce\Module;
+use TEC\Tickets\Emails\Email_Abstract;
+
+/**
+ * Class Handler
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Emails\JSON_LD
+ */
+class Handler {
+
+	/**
+	 * Get the JSON LD data for the email.
+	 *
+	 * @since TBD
+	 *
+	 * @param Email_Abstract $email The email object.
+	 *
+	 * @return array The JSON LD data.
+	 */
+	public static function get_data( \TEC\Tickets\Emails\Email_Abstract $email ): array {
+
+		$data = [];
+
+		if ( $email->slug === 'completed-order'
```

Comment:
We can probably compare against the slug coming from the `Email\Completed_Order` and `Email\Purchase_Receipt` classes so we avoid having a string here, and we're sure that's coming from the class. I don't see us changing the slug, but I'd do it that way.